soccer/gameplay/tactics/wall.py

```diff
@@ -16,24 +16,25 @@ class State(enum.Enum):
         shot = 2
         scramble = 3
 
+    ##
+    # @param num_defenders: number of robots in the wall (default 3)
+    # @param curvature: 'curvature' (in radians) of the wall
```

Comment:
Could you explain what this actually means? Curvature should be in 1 / m

Reply:
*shrug*